MonoDevelop.MSBuild/Schema/BuiltInSchema.cs

```diff
@@ -44,6 +44,7 @@ static MSBuildSchema Load (BuiltInSchemaId[] schemaIds, out IList<MSBuildSchemaL
 		   BuiltInSchemaId.CSharpWarningCodes,
 		   BuiltInSchemaId.AnalyzerWarningCodes,
 		   BuiltInSchemaId.StyleRuleCodes], null, "Microsoft.CSharp.CurrentVersion.targets"),
+		([ BuiltInSchemaId.ILCompiler ], null, "Microsoft.NETCore.Native.targets"),
```

Comment:
I took this name on the basis it was [this file](https://github.com/dotnet/runtime/blob/main/src/coreclr/nativeaot/BuildIntegration/Microsoft.NETCore.Native.targets) that should be used.